### DISCUSSION TITLE
Send logs to Apache Skywalking

### DISCUSSION THREAD

**Discussion by joacatito** [SkyAPM/SkyAPM-dotnet#567] [Q&A] [answered]
Dear all.

I have Skywalking running and I am receiving the trace information, works great!
What I am missing are text logs. They appear under the "Logs" tab. We have nothing there.
I could not find a way to send this kind of logs to Skywalking. Could not find an example too.
There is a way to do it?

Thanks in advance.

**Comment by wu-sheng** [comment, marked as answer]
I can see there is a logging integration at here, https://github.com/SkyAPM/SkyAPM-dotnet/tree/main/sample/SkyApm.Sample.Logging

@liuhaoyang Do we have a doc about this?

**Reply by joacatito** [reply]
Thank you!

I will do some tests and update this thread to help somebody else.
